chef/cookbooks/network/templates/default/suse-cfg.erb

```diff
@@ -59,7 +59,7 @@ ETHERDEVICE=<%=quote(iface["parent"])%>
    # Settings specific to bond devices (ifcfg-bonding(5))
    when @nic.kind_of?(Nic::Bond) -%>
 BONDING_MASTER=yes
-BONDING_MODULE_OPTS=<%=quote("mode=#{@nic.mode} miimon=#{@nic.miimon}")%>
+BONDING_MODULE_OPTS=<%=quote("mode=#{@nic.mode} miimon=#{@nic.miimon} xmit_hash_policy=#{@nic.xmit_hash_policy}")%>
```

Comment:
I have modified the earlier code to also set miimon and xmit_hash_policy on the nic object, if @rhafer thinks that is fine, then it also solves this problem as well (which I've tested and confirmed)